R/browse_url.R

```diff
@@ -0,0 +1,14 @@
+browse_url <- function(url) {
+  translated_url <- tryCatch(
+    {
+      # A local URL needs to be translated to an external one by rstudio.cloud
```

Comment:
To be clear, rstudio.cloud is only one example.
The same behavior occurs on rsp.rstudioservices.com, and I would expect the same in any case where the RStudio IDE is running as a server.